configs/herkin.default.config.js

```diff
@@ -0,0 +1,36 @@
+const path = require('path')
+
+const {
+  DOC_APP_PATH,
+  HERKIN_TESTS_ROOT,
+  HERKIN_FEATURES_DIR,
+  HERKIN_STEPS_DIR,
+  HERKIN_SUPPORT_DIR,
+  HERKIN_UNIT_DIR,
+  HERKIN_WAYPOINT_DIR,
+} = process.env
+
+const rootDir = path.join(__dirname, '../')
+const dockerTestsRoot = path.join(rootDir, 'tests')
+const hostTestsRoot = HERKIN_TESTS_ROOT || path.join(rootDir, 'tests')
+
+module.exports = {
+  paths: {
+    rootDir,
+
+    // if DOC_APP_PATH is defined, we are in the docker container, so look for tests root at <herkin-root>/tests
+    testsRoot: DOC_APP_PATH 
```

Comment:
When inside the container, `hostTestsRoot` is set to the value of the env `HERKIN_TESTS_ROOT`. But `HERKIN_TESTS_ROOT` is an absolute path to the root tests folder **on the host machine** (example: `/Users/michael.carolin/keg-hub/taps/keg-herkin/example/tests`). So we cannot use it in the container, because that path will not exist.